jsapp/js/utils.ts

```diff
@@ -123,6 +123,14 @@ export function formatDate(
   return myMoment.format(format);
 }
 
+/**
+ * Takes a Unix timestamp. Returns a UTC string
+ */
+export function convertUnixTimestampToUtc(time: number): string {
+  let myMoment = moment.unix(time);
```

Comment:
Should we use moment on new code, it's [devs recommend against it](https://momentjs.com/docs/#/-project-status/)